pages/docs/community/styleguide/grammar.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Grammar
+description: This page explains how to use Grammar when writing docs
+---
+#Grammar
```

Comment:
```suggestion
## Grammar
```

Please add introduction content to the first header, introducing the reader to this `Grammar` Style Guide document.